.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1 +1,18 @@
-distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.9.4/apache-maven-3.9.4-bin.zip
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
drop this license header

Reply:
This is what `mvn wrapper:wrapper` generates. It might make it easier to upgrade in the future.